pallets/parachain-staking/src/lib.rs

```diff
@@ -51,7 +51,7 @@
 
 mod delegation_requests;
 pub mod inflation;
-pub mod migrations;
+pub mod migrations; // TODO
```

Comment:
Have `todo` been finished?

Reply:
So has it been finished or not? If yes, remove TODO